src/components/ProfileValidation/TermsAndConditionsWrapped.tsx

```diff
@@ -1,41 +1,62 @@
-import { Box } from "@mui/material"
-import TermsAndConditions from 'synapse-react-client/dist/containers/TermsAndConditions';
+import { Box } from '@mui/material'
+import TermsAndConditions from 'synapse-react-client/dist/containers/TermsAndConditions'
 
-import React from "react"
-import theme from "style/theme"
+import React from 'react'
+import theme from 'style/theme'
+import { ContinueButton } from './ContinueButton'
+import { VerificationSubmission } from 'synapse-react-client/dist/utils/synapseTypes'
 
-const TermsAndConditionsWrapped: React.FC<{ onFormChange: (a: boolean) => void }> = ({ onFormChange }) => (
-    <Box sx={{
+export type TermsAndConditionsWrappedProps = {
+  verificationSubmission: VerificationSubmission
+  onNext: (verificationSubmission: VerificationSubmission) => void
+}
+
+const TermsAndConditionsWrapped: React.FC<TermsAndConditionsWrappedProps> = ({
+  onNext,
+  verificationSubmission,
+}) => {
+  const [isFormComplete, setIsFormComplete] = React.useState(false)
+
+  return (
+    <Box
+      sx={{
         '&  ul': { margin: theme.spacing(0, -8) },
```

Comment:
this is probably a temporary solution until we have a firm design. Using specificity to shoehorn the current component into general design theme